background/index.js

```diff
@@ -32,7 +33,6 @@ chrome.runtime.onConnect.addListener((port) => {
     return;
   }
 
-  // TODO [ToDr] Use connection without elevated privileges here!
   if (port.name === 'web3') {
     port.onMessage.addListener(web3Message(port));
```

Comment:
Since moving handlers to separate files, we might want to do the same for this `web3Message` handler.